src/man/sss-certmap.5.xml

```diff
@@ -723,7 +723,7 @@
                         specific component.
                     </para>
                     <para>
-                        A different component can it either selected by
+                        A different component can be either selected by
```

Comment:
I believe "either" should be after "selected by" here.   Can you change that as well?

So I think it should read something like:
"A different component can be selected by either attribute name, ... or by position, ..."